models/common.py

```diff
@@ -404,9 +404,10 @@ def wrap_frozen_graph(gd, inputs, outputs):
     def forward(self, im, augment=False, visualize=False, val=False):
         # YOLOv5 MultiBackend inference
         b, ch, h, w = im.shape  # batch, channel, height, width
-        if self.pt or self.jit:  # PyTorch
-            y = self.model(im) if self.jit else self.model(im, augment=augment, visualize=visualize)
-            return y if val else y[0]
+        if self.pt:  # PyTorch
```

Comment:
Confirming that this won't get called during training?

Reply:
This is a more general fix plucked from upstream yolov5. It will affect all scripts and take care of a pretty rare edge case. The actual fix for the detect issue is the one-liner in export.py. I should mentioned that in the description